ui/app/clinical/dashboard/controllers/patientDashboardController.js

```diff
@@ -25,29 +25,10 @@ angular.module('bahmni.clinical')
                     alert("We have a full fledged problem");
                 }
             };
-
             $scope.ipdDashboard = {
                 hostData: {
-                    patient: {
-                        uuid: "--- DUMMY UUID FOR TESTING FROM HOST ---"
-                    }
-                },
-                hostApi: {
-                    onConfirm: function () {
-                        const dialogScope = {
-                            message:
-                                "This is a dialog triggered on the host in response to an event from IPD ",
-                            okay: function (close) {
-                                close();
-                            }
-                        };
-
-                        confirmBox({
-                            scope: dialogScope,
-                            actions: [{ name: "okay", display: "Okay" }],
-                            className: "ngdialog-theme-default"
-                        });
-                    }
+                    patientId: $stateParams.patientUuid,
+                    forDate: new Date().toUTCString()
```

Comment:
Same as abve

Reply:
We are directly passing the epoch value of date from bahmniapps. In IPD-frontend repo we are directly receiving the epoch value for forDate

Reply:
Okay, for now lets keep it like this, We might have to revisit this when navigation card is played.